Sources/ArgumentParser/Parsing/Interactive.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  File.swift
+//
+//
+//  Created by Keith  on 2022/6/21.
+//
+
+extension CommandParser {
+  func canInteract(error: Error, split: inout SplitArguments) -> Bool {
+    if let error = error as? ParserError {
+      switch error {
+      case let .missingValueForOption(inputOrigin, name):
```

Comment:
Can we use `guard` for a requirement like this? You can also use `case let` pattern matching in an `if` or `guard` statement instead of a `switch` when you're just looking for one case.

Reply:
done